CompassTests/Shared/TestCompass.swift

```diff
@@ -24,7 +24,7 @@ class TestCompass: XCTestCase {
     let expectation = self.expectationWithDescription("Parse arguments")
     let url = NSURL(string: "compassTests://profile:testUser")!
 
-    Compass.parse(url) { route, arguments in
+    Compass.parse(url) { route, arguments, _ in
```

Comment:
Well, you could make two methods for `parse`, like `Cartography` handles it but that feels a bit odd doesn't it?